languages/fsharp/exercises/concept/basics/.meta/Example.fs

```diff
@@ -2,9 +2,11 @@ module Basics
 
 let expectedMinutesInOven = 40
 
-let remainingMinutesInOven actualMinutesInOven = expectedMinutesInOven - actualMinutesInOven
+let remainingMinutesInOven actualMinutesInOven =
+    expectedMinutesInOven - actualMinutesInOven
 
 let preparationTimeInMinutes numberOfLayers = numberOfLayers * 2
 
-let totalTimeInMinutes numberOfLayers actualMinutesInOven =
-    preparationTimeInMinutes numberOfLayers + actualMinutesInOven
+let elapsedTimeInMinutes numberOfLayers actualMinutesInOven =
+    preparationTimeInMinutes numberOfLayers
+    + actualMinutesInOven
```

Comment:
Once again this formatting feels a little strange, but if that's what the tool which is aligned with the F# style guides suggests than it looks good to me!

Reply:
Yeah, it is a bit weird, I admit. I'll update to the latest version sometime soon and see what that will do.